webpages/api/admin/module_model.php

```diff
@@ -64,30 +96,30 @@ public function getDescriptorClass() {
 
     public static function findAll($db) {
         $query = <<<EOD
-        SELECT id, name, package_name, description, is_enabled
-             FROM module M
-           ORDER BY M.name, M.id;
-EOD;
+            SELECT id, name, package_name, description, is_enabled
+            FROM module M
+            ORDER BY M.name, M.id;
+        EOD;
         $stmt = mysqli_prepare($db, $query);
-        return PlanzModule::processSqlStatement($stmt,$query);
+        return self::processSqlStatement($stmt, $query);
     }
 
     public static function findAllEnabledModules($db) {
         $query = <<<EOD
-        SELECT id, name, package_name, description, is_enabled
-          FROM module M
-         WHERE is_enabled = 1;
-EOD;
+            SELECT id, name, package_name, description, is_enabled
+            FROM module M
+            WHERE is_enabled = 1;
+        EOD;
         $stmt = mysqli_prepare($db, $query);
-        return PlanzModule::processSqlStatement($stmt,$query);
+        return self::processSqlStatement($stmt, $query);
     }
 
     public function updateEnabled($db, $value) {
         $query = <<<EOD
-           UPDATE module M
-              SET is_enabled = ?
+            UPDATE module M
+            SET is_enabled = ?
             WHERE id = ?;
-EOD;
+        EOD;
```

Comment:
I think Leanne originally told me to keep the EOD aligned to the left because of reasons (otherwise it was a problem for some version of... um.. PHP? Or something).

Reply:
Yes, in PHP < 7.3, the closing identifier must be left aligned. 

In PHP >= 7.3 the closing identifier can be indented, but must not be indented further than any line of text in the string. The same number of leading spaces get removed from each line of the string.

If we are allowing arrow functions, which require PHP 7.4, then indenting closing string identifiers aren't an issue.

Reply:
Okay. Good to know.

Reply:
I'll update the Readme to specify minimum version 7.3.